isis/tests/AlternativeTestCsmModel.cpp

```diff
@@ -65,11 +91,21 @@ std::string AlternativeTestCsmModel::getReferenceDateAndTime() const {
 }
 
 std::string AlternativeTestCsmModel::getModelState() const {
-  return m_modelState;
+  json state;
+  state["test_param_one"] = m_param_values[0];
+  state["test_param_two"] = m_param_values[1];
+  state["test_param_three"] = m_param_values[2];
+  state["test_param_four"] = m_param_values[3];
+  return AlternativeTestCsmModel::SENSOR_MODEL_NAME + "\n" + state.dump();
 }
 
 void AlternativeTestCsmModel::replaceModelState(const std::string& argState) {
-  m_modelState = argState;
+  // Get the JSON substring
```

Comment:
Technically, we should also check that the string before the new line matches the model name, but I think that's a bit too far.